go/tasks/plugins/k8s/kfoperators/common/common_operator.go

```diff
@@ -94,27 +131,30 @@ func GetLogs(taskType string, name string, namespace string,
 		}
 		taskLogs = append(taskLogs, workerLog.TaskLogs...)
 	}
-	// get all parameter servers logs
-	for psReplicaIndex := int32(0); psReplicaIndex < psReplicasCount; psReplicaIndex++ {
-		psReplicaLog, err := logPlugin.GetTaskLogs(tasklog.Input{
-			PodName:   name + fmt.Sprintf("-psReplica-%d", psReplicaIndex),
-			Namespace: namespace,
-		})
-		if err != nil {
-			return nil, err
+
+	if taskType != MPITaskType && taskType != PytorchTaskType {
```

Comment:
```suggestion
	if taskType == MPITaskType || taskType == PytorchTaskType {
            return taskLogs, nil
      }
```

so you don't have to indent the whole block below